store/driver/util_test.go

```diff
@@ -1,82 +0,0 @@
-// Copyright 2021 PingCAP, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package driver
-
-import (
-	"context"
-	"flag"
-	"fmt"
-	"testing"
-
-	. "github.com/pingcap/check"
-	"github.com/pingcap/errors"
-	"github.com/pingcap/tidb/kv"
-	"github.com/pingcap/tidb/store/copr"
-	"github.com/pingcap/tidb/store/mockstore/unistore"
-	"github.com/tikv/client-go/v2/tikv"
-)
-
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	tikv.EnableFailpoints()
-	TestingT(t)
-}
-
-var (
-	pdAddrs  = flag.String("pd-addrs", "127.0.0.1:2379", "pd addrs")
-	withTiKV = flag.Bool("with-tikv", false, "run tests with TiKV cluster started. (not use the mock server)")
-)
```

Comment:
@disksing @XuHuaiyu I don't see any where to config this variable, and thus remove them just respect the default.

Reply:
I think Ci environment will use it to test with a real tikv server.

Reply:
@zhouqiang-cl I remember that won't be one, at least for now. Please confirm.

Reply:
OK that may make sense, let me try to confirm it and if true, update the code.

Reply:
confirmed https://github.com/PingCAP-QE/ci/blob/bcb0c98ff3abfbf0a1e1ab8dc60ea280ef95d1ee/jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy#L132

Updating....

Reply:
Updated. @disksing PTAL.